### ISSUE TITLE
Possible to compile for ARM?

### ISSUE THREAD

**Issue by tracerrx** [ishiDACo/vulsrepo#194]
On a RPI 4
`pi@bon-vuls:~/vulsrepo/server $ ./vulsrepo-server
-bash: ./vulsrepo-server: cannot execute binary file: Exec format error

pi@bon-vuls:~/vulsrepo/server $ file vulsrepo-server
vulsrepo-server: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=d21d01d420ef7a83472ce6c824e6965d3a1c0df9, with debug_info, not stripped
pi@bon-vuls:~/vulsrepo/server $`

**Comment by tracerrx** [comment]
Fixed.... Solution for others:
sudo apt-get install golang

go get github.com/BurntSushi/toml

go get github.com/abbot/go-http-auth

go run ./main.go

**Comment by necrose99** [comment]
Yes , Vulsrepo , can be easily compiled from [GOX](https://github.com/mitchellh/gox).  for multiple targets , Windows arm64/amd64 , BSD/Macos  (https://github.com/goreleaser/goreleaser) could be setup to make exe,rpm, tarballs , etc of Vulsrepo... 

might also already be packaged in Kali/Parrotsec.   @Pentoo / Gentoo needs some updated to ebuild scripts to bring more current.
